saleor/graphql/account/tests/test_account.py

```diff
@@ -504,6 +504,267 @@ def test_query_staff_user_with_orders_and_manage_orders_perm(
     }
 
 
+def test_query_customer_user_orders_by_staff_with_manage_users(
+    staff_api_client,
+    customer_user,
+    order_list,
+    permission_manage_users,
+):
+    # given
+    query = """
+        query User($id: ID!) {
+            user(id: $id) {
+                email
+                orders(first: 10) {
+                    totalCount
+                    edges {
+                        node {
+                            id
+                            number
+                            subtotal {
+                                gross {
+                                    amount
+                                    currency
+                                }
+                            }
+                            total {
+                                gross {
+                                    amount
+                                    currency
+                                }
+                            }
+                            totalAuthorized {
+                                amount
+                            }
+                            undiscountedTotal {
+                                gross {
+                                    amount
+                                }
+                            }
+                            totalBalance {
+                                amount
+
+                            }
+                            isPaid
+                            paymentStatus
+                            paymentStatusDisplay
+                            statusDisplay
+                            userEmail
+                            languageCodeEnum
+
+                        }
+                    }
+                }
+            }
+        }
+    """
+    order_unfulfilled = order_list[0]
+    order_unfulfilled.user = customer_user
+
+    order_unconfirmed = order_list[1]
+    order_unconfirmed.status = OrderStatus.UNCONFIRMED
```

Comment:
we set the status expliciltly for this one and draft but not for unfulfilled, could be added for consistency.